### ISSUE TITLE
Keyboard navigation unreachable dashboard sections

### ISSUE THREAD

**Issue by filiptronicek** [gitpod-io/gitpod#16724]
### Bug description

In the dashboard, there are many sections and individual components currently not accessible by sequential keyboard navigation. This means it's hard and sometimes impossible for users using screen readers to interface with Gitpod beyond creating a workspace.

## Affected components
- https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/menu/OrganizationSelector.tsx (when closed, while it is open, it's inner options are focusable)
- [the feedback button in the menu](https://github.com/gitpod-io/gitpod/blob/cac068f6ce862ffaf9ecb2f68640d68c46121c9a/components/dashboard/src/menu/Menu.tsx#L190-L193)
- [`UserMenu`](https://github.com/gitpod-io/gitpod/blob/cac068f6ce862ffaf9ecb2f68640d68c46121c9a/components/dashboard/src/menu/Menu.tsx#L114-L120)

## Further reading

- https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets
- https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

### Steps to reproduce

f

**Comment by gtsiolis** [comment]
Thanks for collecting these issues, opening this, and using the https://github.com/gitpod-io/gitpod/labels/aspect%3A%20accessibility label, @filiptronicek!
